modules/kubernetes-addons/helm_addon/locals.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  service_account_name = "${var.name}-sa"
+  namespace            = "${var.name}-ns"
+
+  default_helm_config = {
+    name                       = var.name
+    chart                      = var.name
+    namespace                  = local.namespace
+    timeout                    = "1200"
+    create_namespace           = true
+    description                = "Kubernetes Helm AddOn"
+    lint                       = false
+    wait                       = true
+    wait_for_jobs              = false
+    verify                     = false
+    keyring                    = ""
+    repository_key_file        = ""
+    repository_cert_file       = ""
+    repository_ca_file         = ""
+    repository_username        = ""
+    repository_password        = ""
+    disable_webhooks           = false
```

Comment:
if these are all the default values in the helm provider, don't think we need to include them in default config.